hedera-node/hedera-util-service-impl/src/main/java/com/hedera/node/app/service/util/impl/handlers/AtomicBatchHandler.java

```diff
@@ -81,6 +104,10 @@ public void pureChecks(@NonNull final PureChecksContext context) throws PreCheck
             if (!txBody.hasBatchKey()) {
                 throw new PreCheckException(MISSING_BATCH_KEY);
             }
+
+            if (!txBody.nodeAccountID().equals(ATOMIC_BATCH_NODE_ACCOUNT_ID)) {
+                throw new PreCheckException(INVALID_NODE_ACCOUNT_ID);
```

Comment:
I am assuming this is hard coded value that need to be checked with shard and realm set to 0. or should we use the config shard and realm ?